sanitizer/_size.js

```diff
@@ -1,36 +1,54 @@
 const _ = require('lodash');
-const MIN_SIZE = 1;
-const MAX_SIZE = 40;
-const DEFAULT_SIZE = 10;
+const peliasConfig = require('pelias-config').generate().api;
+//Allow custom min, max and default sizes
+const MIN_SIZE = peliasConfig.minSize || 1;
+const MAX_SIZE = peliasConfig.maxSize || 40;
+let default_size_init;
+if (peliasConfig.DEFAULT_SIZE) {
+  //Cant be smaller/bigger than min/max due to validation in schema.js
+  default_size_init = peliasConfig.DEFAULT_SIZE;
+}
+else if (MIN_SIZE > 10) {
+  default_size_init = MIN_SIZE;
+}
+else if (MAX_SIZE < 10) {
+  default_size_init = MAX_SIZE;
+}
+else {
+  default_size_init = 10;
+}
+const DEFAULT_SIZE =default_size_init;
+
+
 
 // validate inputs, convert types and apply defaults
-function _setup( size_min, size_max, size_def ){
+function _setup(size_min, size_max, size_def) {
 
   // allow caller to inject custom min/max/default values
-  if( !_.isFinite( size_min ) ){ size_min = MIN_SIZE; }
-  if( !_.isFinite( size_max ) ){ size_max = MAX_SIZE; }
-  if( !_.isFinite( size_def ) ){ size_def = DEFAULT_SIZE; }
+  if (!_.isFinite(size_min)) { size_min = MIN_SIZE; }
+  if (!_.isFinite(size_max)) { size_max = MAX_SIZE; }
+  if (!_.isFinite(size_def)) { size_def = default_size_init; }
```

Comment:
this can remain the same and continue using the constant, right?

Reply:
Yeah, artifact from search and replace. Gonna change it.